docs/content/src/learn/tutorials/saas-integrations/amazons3/working-with-s3-bucket/src/guide/Module.md

```diff
@@ -0,0 +1 @@
+# Working with Amazon S3 Bucket
```

Comment:
We need to update the Module.md as well.

Reply:
Fixed